sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -1420,6 +1432,21 @@ def _more_tags(self):
     check_estimator_tags_renamed("OkayEstimator", OkayEstimator())
 
 
+def test_check_classifier_not_supporting_multiclass():
+    """Check that when the estimator has the wrong tags.classifier_tags.multi_class
+    set, the test fails."""
+
+    class BadEstimator(BaseEstimator):
+        # we don't actually need to define the tag here since we're running the test
+        # manually.
```

Comment:
```suggestion
        # we don't actually need to define the tag here since we're running the test
        # manually, and BaseEstimator defaults to multi_output=False
```